.pre-commit-config.yaml

```diff
@@ -0,0 +1,41 @@
+ci:
+  skip:
+    - eslint
+    - fmt
+    - clippy
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      - id: check-yaml
+      - id: check-json
+      - id: check-toml
+      - id: check-xml
+      - id: check-symlinks
+      - id: end-of-file-fixer
+        exclude: \.(svg|png)$
+      - id: trailing-whitespace
+        exclude: \.(svg|png)$
+      - id: check-added-large-files
+      - id: check-merge-conflict
+      - id: check-shebang-scripts-are-executable
+      - id: detect-private-key
+      - id: forbid-new-submodules
+      - id: mixed-line-ending
+      - id: no-commit-to-branch
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v2.7.1
+    hooks:
+      - id: prettier
+  - repo: https://github.com/pre-commit/mirrors-eslint
+    rev: v8.23.1
+    hooks:
+      - id: eslint
+  - repo: https://github.com/doublify/pre-commit-rust
```

Comment:
make sure to dial clippy up to 11 (`pedantic nursery unwrap_used`)

Reply:
Thanks, will set that now.